_locales/pl/messages.json

```
@@ -0,0 +1,383 @@
+		"message": "Wystawia przedmiot za najwyższą podaną cenę zlecenia kupna\n\nWyświetlana cena to kwota, którą otrzymasz (bez opłat)."
+	},
+	"widget_view_app": {
+		"message": "Wyświetl aplikację $1 na SteamDB"
+	},
+	"widget_view_sub": {
+		"message": "Wyświetl paczkę $1 on SteamDB"
+	},
+	"options": {
+		"message": "Opcje"
+	},
+	"options_welcome_title": {
+		"message": "👋 Dzięki za zainstalowanie rozszerzenia SteamDB!"
+	},
+	"options_welcome_description": {
+		"message": "Przejrzyj poniższe opcje i ustaw te, które preferujesz. Aby wrócić tutaj w przyszłości:"
+	},
+	"options_welcome_option1": {
+		"message": "Strona ustawień na SteamDB"
+	},
+	"options_welcome_option2": {
+		"message": "Rozwijane menu konta na stronach Steam"
+	},
+	"options_welcome_option3": {
+		"message": "Strona rozszerzeń twojej przeglądarki"
+	},
+	"options_review": {
+		"message": "Wystaw recenzje dla tego rozszerzenia"
+	},
+	"options_permissions_description": {
+		"message": "To rozszerzenie wymaga dodatkowych uprawnień do poprawnego działania w SteamDB, Sklepie Steam i Społeczności Steam."
+	},
+	"options_permissions_button": {
+		"message": "⚠️ Przyznaj uprawnienia"
+	},
+	"options_footer1": {
+		"message": "Stworzone z 💙 przez zespół <a href=\"https://steamdb.info\" target=\"_blank\">SteamDB</a>"
+	},
+	"options_footer2": {
+		"message": "To rozszerzenie jest <a href=\"https://github.com/SteamDatabase/BrowserExtension\" target=\"_blank\">otwarto-źrodłowe</a>, dziękuję wszystkim zaangażowanym!"
+	},
+	"options_disabled_checkboxes": {
+		"message": "* Wyłączone pola wyboru oznaczają wbudowane funkcje."
+	},
+	"options_header_website": {
+		"message": "Strona SteamDB"
+	},
+	"options_header_extra_data": {
+		"message": "Dodatkowe dane"
+	},
+	"options_header_buttons_and_links": {
+		"message": "Przyciski i odnośniki"
+	},
+	"options_header_inventory": {
+		"message": "Ekwipunek"
+	},
+	"options_header_trade_offers": {
+		"message": "Oferty wymiany"
+	},
+	"options_header_enhancements": {
+		"message": "Rozszerzenia"
+	},
+	"options_website_explanation": {
+		"message": "Wyjaśnienie"
+	},
+	"options_website_explanation1": {
+		"message": "Gdy ta opcja jest włączona, rozszerzenie zażąda niektórych danych użytkownika (lista życzeń, posiadane, śledzone, ignorowane produkty i produkty w koszyku) ze <a href=\"https://store.steampowered.com/dynamicstore/userdata/\" target=\"_blank\">Sklepu Steam</a> i użyje ich do pokazania wyróżnień na stronie."
+	},
+	"options_website_explanation2": {
+		"message": "Aby to działało, musisz być zalogowany w sklepie."
+	},
+	"options_website_explanation3": {
+		"message": "Jest to w całości zawarte w twojej przeglądarce i nigdy nie jest wysyłane do bazy SteamDB."
+	},
+	"options_website_explanation4": {
+		"message": "Aby zapobiec nadmiernym zapytaniom do Sklepu Steam, dane użytkownika są buforowane w lokalnej pamięci przeglądarki."
+	},
+	"options_website_highlight": {
+		"message": "Wyróżnij posiadane i pożądane produkty na steamdb.info"

```

```suggestion
		"message": "Wyróżnij posiadane i znajdujące się na liście życzeń produkty na steamdb.info"
```

Tak, to jest dużo dłuższe niż oryginał, "pożądane" nie odpowiada "wished", nikt w przeciwnym razie nie będzie wiedział o co chodzi, skrót typu "życzone sobie" jest okropny.